documentation/Ingress-controller-upgrade.md

```diff
@@ -37,3 +37,4 @@ make test terraform-apply CONFIRM_TEST=yes
         ```
 - Follow the same process for production and raise a PR for each environment.
 - Wait at least 24 hours before applying upgrade to production (if any downtime was obversed during deployment(s) to lower environment(s), notify the application teams before production upgrade).
+- After a successful upgrade, check metrics are still displaying and working well in Grafana dashboards: https://grafana.teacherservices.cloud
```

Comment:
Monitoring should be checked for each environment. This step should be before "Follow the same process" and show the different environment grafana URLs